### PR TITLE
Fixes #32911 - Create content view button fix

### PR DESCRIPTION
**Pull request by Andrewgdewar**
Steps to reproduce:

1. On the new CV page, click the Create content view button
2. Fill out the form and click Create
3. It takes you to the Versions tab for the newly created CV
4. Click 'Content Views' in the breadcrumb
5. Click the Create content view button again

Expected: The Create Content View modal shows.

### CODE REVIEW

**Comment by jeremylenz** [comment]
[test katello]

File: webpack/scenes/ContentViews/Create/CreateContentViewForm.js [hunk 1]
```diff
@@ -29,13 +29,13 @@ const CreateContentViewForm = ({ setModalOpen }) => {
...
     const { id } = response;
-    if (id && status === STATUS.RESOLVED) {
+    if (id && status === STATUS.RESOLVED && saving) {
       setSaving(false);
       setRedirect(true);
     } else if (status === STATUS.ERROR) {
       setSaving(false);
     }
-  }, [response, status, error]);
+  }, [JSON.stringify(response), status, error, saving]);
```
Comment (jeremylenz) [inline webpack/scenes/ContentViews/Create/CreateContentViewForm.js:38]: Now that we use `useDeepCompareEffect` we don't need to `JSON.stringify` anymore :)

Comment (jeremylenz) [reply webpack/scenes/ContentViews/Create/CreateContentViewForm.js:38]: (You might need to rebase the PR to the latest master, I think that was just merged in.)

Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Create/CreateContentViewForm.js:38]: I will reach out to do this! thanks

**Comment by jeremylenz** [comment]
[test katello]

**Review by jeremylenz — APPROVED** [review]
:heavy_check_mark: Works as expected

Thanks @Andrewgdewar !  APJ

**Comment by jeremylenz** [comment]
[test katello]

**Comment by jeremylenz** [comment]
[test katello]

**Pull request merged**